docs/guide/index.md

```diff
@@ -1,14 +1,79 @@
 ---
-title: 介绍
+title: Introducation
 order: 1
 toc: menu
 nav:
-  title: 指南
+  title: Guide
   order: 1
 ---
 
-## 什么是 dumi？
+## What's dumi?
 
-## 特性
+dumi, pronounced like [duːmɪ], is a documentation tool for React component developing libraries. It can provide one-stop component development experience for developer with [father](https://github.com/umijs/father), in which **father for building and dumi for component development & writting docs**.
```

Comment:
```suggestion
dumi, pronounced like [duːmɪ], is a documentation tool for developing libraries. It can provide one-stop component development experience for developer with [father](https://github.com/umijs/father), in which **father for building and dumi for component development & writting docs**.
```